django_sites_extensions/tests/test_middleware.py

```diff
@@ -55,3 +55,16 @@ def test_redirects_cached(self):
         request = self._make_request('/bar')
         with self.assertNumQueries(0):
             self.middleware.process_request(request)
+
+    def test_new_style_middleware_compatibility(self):
```

Comment:
As written, isn't this just a copy of the test above?

Reply:
removing now.